Getting an error when I run make watch:

```
[vagrant]$ make watch
godep go install ./...
LIVEPLANTDEBUG=1 reflex -r '\.go -s -- sh -c 'make run'
/bin/sh: 1: Syntax error: Unterminated quoted string
make: *** [watch] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -20,7 +20,7 @@ run: $(TARGET)
 	forego start
 
 watch: $(TARGET)
-	LIVEPLANTDEBUG=1 reflex -r '\.go$' -s -- sh -c 'make run'
+	LIVEPLANTDEBUG=1 reflex -r '\.go$$' -s -- sh -c 'make run'
 
 clean: 
 	rm $(TARGET)

--- a/README.md
+++ b/README.md
@@ -18,8 +18,6 @@ Liveplant server is a REST API for [liveplant.io][]
 - `make run` will compile and run your code. Your server will be available at
   [localhost:5000](http://localhost:5000).
 - `make fmt` will format your code with gofmt.
-- `make watch` will watch your code for changes and recompile and relaunch the
-  server.
 
 ## Usage
 

